haystack/utils/deepsetcloud.py

```diff
@@ -865,9 +865,50 @@ def _get_evaluation_sets(self, workspace: Optional[str] = None) -> Generator:
         evaluation_set_url = f"{url}/evaluation_sets"
         return self.client.get_with_auto_paging(url=evaluation_set_url)
 
-    def _get_evaluation_set(
+    def upload_evaluation_set(self, file_path: Path, workspace: Optional[str] = None):
+        """
+        Uploads an evaluation set file.
+        The evaluation set will be accessible under the name of the uploaded file.
+
+        Currently only CSV files (having "," as delimiter) with the following columns are supported:
+        - question (or query): the question or query of the label
+        - text: the answer or relevant text to the question or query
+        - context: the surrounding words of the text (should be more than 100 characters)
+        - file_name: the name of the file within the workspace that contains the text
```

Comment:
it would be so much easier to understand if we could just say: The name of the file that contains the answer